crates/nu-command/src/commands/config/set.rs

```diff
@@ -93,6 +94,10 @@ pub async fn set(args: CommandArgs) -> Result<OutputStream, ShellError> {
             ..
         }) => {
             config::write(&changes.entries, &path)?;
+            ctx.reload_config(&ConfigPath::Global(
```

Comment:
We now have to reload the config explicitly in `config set` after making changes to it.

Reply:
(path will always be Some() because of above logic)